### PR TITLE
build(website): use vite-react-ssg

### PR DESCRIPTION
**Pull request by Daydreamer-riri**

### CODE REVIEW

**Comment by childrentime** [comment]
@Daydreamer-riri HI, it breaks netlify and lint

**Comment by Daydreamer-riri** [comment]
I got it. I will fix these problems tomorrow.

**Comment by childrentime** [comment]
@Daydreamer-riri It seems file. and we need remove the comment in css.

File: packages/website/package.json
Comment (childrentime) [inline packages/website/package.json:1]: lint

**Review by childrentime — CHANGES_REQUESTED** [review]

**Review by childrentime — DISMISSED** [review]

**Comment by childrentime** [comment]
@Daydreamer-riri Are there any other changes need to commit?

**Comment by Daydreamer-riri** [comment]
That's all.

**Review by childrentime — APPROVED** [review]

**Pull request merged**